src/lib/ngsi/ContextAttribute.cpp

```diff
@@ -828,10 +848,14 @@ std::string ContextAttribute::toJson
     {
       out += JSON_VALUE_BOOL("value", boolValue);
     }
-    else if (valueType == orion::ValueTypeNone)
+    else if (valueType == orion::ValueTypeNull)
     {
       out += JSON_STR("value") + ":" + "null";
     }
+    else if (valueType == orion::ValueTypeNotGiven)
+    {
+      LM_E(("Runtime Error (ValueTypeNotGiven in compound value)"));
```

Comment:
Not sure how we would end up here ... 
This is probably dead code, but to be safe, let's keep it.

The text in English would be:

    No value in compound value :-)

Reply:
Fixed in e85892cdb28f2f510fd2b0a38cacbb43082f9613